MovieApp/PresentationLayer/Components/OptionBar/OptionBar+Design.swift

```diff
@@ -0,0 +1,100 @@
+import UIKit
+
+extension OptionBar: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
```

Comment:
remove this line, `defineLayoutForViews()` should be called only from `didMoveToSuperview` in classes that extend `UIView`. So, in OptionBar.swift, override that function and call `defineLayoutForViews()`.

Reply:
What's the reasoning for this? Intuitively I think it's so that the view is laid out whenever it's position in the superview is changed but I'm not sure.